src/components/app-configure/access/DidDelegate.vue

```diff
@@ -0,0 +1,209 @@
+<script lang="ts" setup>
+import moment from 'moment'
+
+import SettingCard from '@/components/app-configure/SettingCard.vue'
+import VCard from '@/components/lib/VCard/VCard.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import { useAppsStore } from '@/stores/apps.store'
+import { useAppId } from '@/use/getAppId'
+
+const appId = useAppId()
+const appsStore = useAppsStore()
+
+const app = appsStore.app(appId)
+const delegates = app.access.delegates
+
+function addDelegate() {
+  //
+}
+
+function truncate(address: string) {
+  return address.substring(0, 6) + '....' + address.substr(address.length - 6)
+}
```

Comment:
See if we can move this to a utility function.

Reply:
Resolved here: https://github.com/arcana-network/developer-dashboard/pull/144/commits/2bad363f2d40991a4be2e50b7001832cdef39536